packages/core/lib/commands/test/run.js

```diff
@@ -116,6 +132,11 @@ module.exports = async function (options) {
 
     // configuredNetwork will spread only when it is defined and ignored when undefined
     ganacheOptions = { ...ganacheOptions, port, ...configuredNetwork };
```

Comment:
As an example of what I'm talking about -- and sorry for not catching this on your previous PR! -- the way you set up `ganacheOptions` earlier, and then mutate it here, is confusing; it would be much more readable if it was all set up in one place.  Like, I completely missed that the `configuredNetwork` options get dumped into the Ganache options, because it happens separately later after setup, and not just after but quite a bit after, where I wouldn't think to look for it.  Having variables change their value well after they're set up is generally not something you expect to happen!

That said, that's obviously a separate issue from this PR, so you don't need to change it here.

Reply:
Sure, I will change this thing in another PR.